docs/_posts/2018-02-22-units-of-measurement.md

```diff
@@ -0,0 +1,109 @@
+
+##### UoM data with QuantityType
+Sensor devices and internet services connected to Eclipse SmartHome as Things all provide their scalar values in an implicit unit of measurement. The binding developer knows what exact unit the sensor or service data uses and can now pass those values to the framework using the new [`QuantityType`](../documentation/concepts/units-of-measurement.html). QuantityType represents the scalar value with the corresponding unit. It also serves as the main API to convert between different units.
+The Yahoo Weather binding makes use of QuantityType and publishes state updates in the following form:
+
+```(java)
+QuantityType<Temperature> state = new QuantityType<>(tempDouble, SIUnits.CELSIUS);
+updateState(temperatureChannelUID, state);
+```
+
+For this example the binding developer knows from the Yahoo API description that centigrade is used for temperature values. For pressure values the API description states that Hecto Pascal (hPa) is used, so a state update for the pressure channel looks like this:
+
+```(java)
+QuantityType<Pressure> state = new QuantityType<>(pressDouble, HECTO(SIUnits.PASCAL));
+updateState(pressureChannelUID, state);
+```
+
+In the temperature example the state is marked to be a "<Temperature>" typed QuantitType. It will only accept units which correspond to the dimension type, like centigrade, degree Fahrenheit or Kelvin.
+In the same way the Thing`s Channels and the linked Number items have to be typed to match against each other. Both Channel and Item definition use an extended item type format which passes the expected dimension to a Number item.
+
+The temperature Channel definition for Yahoo Weather looks like this:
+
+```(xml)
+<channel-type id="temperature">
+    <item-type>Number:Temperature</item-type>
+    <label>Temperature</label>
+    <description>Current temperature</description>
+    <category>Temperature</category>
+    <state readOnly="true" pattern="%.1f %unit%"/>
+</channel-type>
+```
+
+A corresponding item definition in the *.items DSL is then:
+
+```
+Number:Temperature Weather_Temperature "Outside Temperature [%.1f %unit%]" { channel="yahooweather:weather:berlin:temperature" }
+```
+
+Both definitions use the extended item type `Number:Temperature` which configures the Number item to only accept state updates with units from the dimension "Temperature".
+For backwards compatibility a Number item without a dimension specified will receive state updates as DecimalType. For this situation the scalar value from the QuantityType is taken without any conversion and passed on to the item. 
+
+##### Conversion
+Once a binding provides state updates with QuantityTypes the unit which should be used on the item can be different from the unit selected by the binding.
+
+###### Locale Conversion
+The default behaviour of the framework is to convert the source value to the unit which is defined to be the default for the current measurement system.
```

Comment:
behaviour: are we using british or american english? I thought we are using the later one, so: behavior